R/oauth-flow-auth-code.R

```diff
@@ -56,9 +56,11 @@ req_oauth_auth_code <- function(req, client,
                                 pkce = TRUE,
                                 auth_params = list(),
                                 token_params = list(),
+                                type = c("desktop", "web"),
```

Comment:
For consistency, maybe we should use type `"hosted"` here instead of type `"web"`.

Reply:
It occurred to me that maybe we could do this automatically based on whether or not the `redirect_url` was to localhost?

Reply:
Let me merge this and then present a concrete proposal.